outbox/shipment-service/src/main/java/io/debezium/examples/outbox/shipment/service/ShipmentService.java

```diff
@@ -18,6 +16,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.fasterxml.jackson.databind.JsonNode;
```

Comment:
Fine for me, but I'm curious why you changed from Jsonb to Jackson?

Reply:
Did I?  The shipment service was mostly left untouched as the Quarkus based shipment-service was already there and it looks like it was using Jackson already.  Since that seems to be the defacto in Quarkus, that's likely why nonetheless.

Reply:
Ah, yes, that will be it. Thanks!